handlers_manager.go

```diff
@@ -127,9 +127,8 @@ func (hm *HandlersManager) RegisterHandler(h IHandler) error {
 
 		hm.handlerVersions[route] = version
 
-		if _, ok := handlerType.MethodByName("V" + strconv.Itoa(v) + "UseCache"); ok {
-			version.UseCache = true
-		}
+		_, version.UseCache = handlerType.MethodByName(handlerMethodPrefix + "UseCache")
+		_, version.AcceptJSON = handlerType.MethodByName(handlerMethodPrefix + "AcceptJSON")
```

Comment:
You should automatically detect can In arguments be sending by URI params or not (flat structure or not). And this logic must be in transport.